handlers/easy_info/__init__.py

```diff
@@ -0,0 +1,12 @@
+from eventhandler import EventHandler
+
+
+class EasyInfoHandler(EventHandler):
+    def on_issue_labeled(self, api, payload):
+        new_label = payload['label']['name']
+
+        if new_label == 'E-Easy':
```

Comment:
I made it lowercase the label now, since this wouldn't be case-insensitive otherwise (the GitHub API already is, apparently).